Tests/scripts/infrastructure_tests/test_lock_cloud_machines.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from Tests.scripts.lock_cloud_machines import get_my_place_in_the_queue, try_to_lock_machine, get_machines_locks_details
+
+
+def test_try_to_lock_machine(mocker):
```

Comment:
this is a good test for when it is not free, let's add a test to when it is free.